hokusai/staging.yml

```diff
@@ -139,3 +139,75 @@ spec:
                     operator: In
                     values:
                     - background
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: opstools-cleanup-pods-by-name
+spec:
+  schedule: "0 * * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 1
```

Comment:
This allows 1 retry. Since the cron runs frequent (hourly), I think it's fine to have no retry (`0`).

Reply:
Thanks, done in https://github.com/artsy/opstools/pull/29/commits/d0a978b9d93d150c5de37eb57ea83cd7030829c8#diff-8ba030368b0c3dfd60ec6bedbbb859e1ca856d64344e6e72b59ffeac4d310d3fR117